packages/components/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 -   `ItemGroup`: Update button focus state styles to be inline with other button focus states in the editor. ([#51576](https://github.com/WordPress/gutenberg/pull/51576)).
 -   `ItemGroup`: Update button focus state styles to target `:focus-visible` rather than `:focus`. ([#51787](https://github.com/WordPress/gutenberg/pull/51787)).
 
+### Experimental
+
+-   `Tabs`: Create a new version of `TabPanel` with updated internals, while maintaining the same functionality and API surface  ([#52133](https://github.com/WordPress/gutenberg/pull/52133)).
```

Comment:
I'm curious about the reasoning for the component naming.

I kind of thought the plan was to:

1. Implement `Tabs`, a new component with a new API and higher composability.
2. Update the internals of `TabPanel` with whatever library we use in `Tabs` to reduce code duplication. API remains the same.

Is there a different plan?